addOns/ascanrulesBeta/src/test/java/org/zaproxy/zap/extension/ascanrulesBeta/HiddenFilesScanRuleUnitTest.java

```diff
@@ -481,6 +481,132 @@ void shouldRaiseAlertIfTestedUrlRespondsOkWithRelevantContentAndAppropriateNotCo
         assertEquals(rule.getReference(), alert.getReference());
     }
 
+    @Test
+    void shouldRaiseAlertIfTestedUrlRespondsOkWithRelevantContentSVNEntries()
```

Comment:
Are these tests really needed? They seem to be testing what was already tested earlier (it might not be the same content but the logic being tested is the same).

Reply:
Heh - just trying to make sure the check triggers less false positives on updates, but I can go either way on the "need"

Reply:
hmmm, how tricky would it be to make them use the file in the test? If it's not hard, I might be up for changing that, else I'm happy to remove.

Reply:
removed these

Reply:
They are still present in the latest commit.

Regarding the previous question, I  think it just needs to call `init()` to load the file.

Reply:
Heh, I missed the git add. should be correct now

Reply:
There are still some?

There's still a commit pending above: https://github.com/zaproxy/zap-extensions/pull/3471#discussion_r788968874

Reply:
It's now under :)